logger/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @copyright (c)2021, Alert Logic, Inc
+ * @doc
+ * 
+ * Logger setup for Lambda functions
+ * @end
+ * 
+ */
+
+const { createLogger, format, transports } = require('winston');
+const { combine, timestamp, errors, json } = format;
+
+const logger = createLogger({
+    level: process.env.LOG_LEVEL || 'info',
+    format: combine(
+        timestamp(),
+        errors({ stack: true }),
```

Comment:
I think we don't need stack for all errors. We only need them for exceptions and for errors which are handles gracefully we normally don't need a stack trace. Is is possible to pass this parameter into `logger.error()`?

Reply:
Let's expose error options through env var as well, like `LOG_ERROR_OPTS`